lib/src/tile.dart

```diff
@@ -1,34 +1,38 @@
-part of tiled;
+import 'dart:math';
+import '../tiled.dart';
+import 'tileset.dart';
+import 'image.dart';
+import 'flips.dart';
 
 class Tile {
   // Tile IDs are 0-based, to conform with TMX documentation.
-  int tileId;
+  int tileId = 0;
   Tileset tileset;
 
   // Tile global IDs aren't 1-based, but start from "1" (0 being an "null tile").
-  int gid;
+  int gid = 0;
 
-  int width;
-  int height;
-  int spacing;
-  int margin;
+  int width = 0;
+  int height = 0;
+  int spacing = 0;
+  int margin = 0;
```

Comment:
These should be late right?

Reply:
Maybe, but there is also `Tile.emptyTile()`, so it is easier, I think.

Reply:
Then the default values for an empty tile should be in the constructor.